server/app/src/main/resources/application.properties

```diff
@@ -1,2 +1,6 @@
 io.delta.sharing.api.server.defaultMaxResults=10
-quarkus.banner.path=banner.txt
\ No newline at end of file
+quarkus.banner.path=banner.txt
+
+whitefox.server.authentication.enabled=false
+whitefox.server.authentication.bearerToken=myToken
```

Comment:
I think these should stay in the test application properties, not the the main one, am I right?

Reply:
I would keep the whitefox.server.authentication.enabled=false as default and remove the token one